core/modules/replica/testNamedMutexRegistry.cc

```diff
@@ -171,6 +172,18 @@ BOOST_AUTO_TEST_CASE(NamedMutexRegistryTest2) {
 
 BOOST_AUTO_TEST_CASE(NamedMutexRegistryTest3) {
 
+    bool testMaxCpuUsage = false;
+    for (int i = 1; i < framework::master_test_suite().argc; ++i) {
+        string const arg = string(framework::master_test_suite().argv[1]);
```

Comment:
is `1` supposed to be `i` here?

Reply:
I'm not sure I understood the question. In C/C++ the `argv` array is zero-based, where the index `[0]` represents a path to the application itself. Actual command-line arguments start with index `1`.